redfish-client/redfish-client

```diff
@@ -406,21 +415,34 @@ if __name__ == '__main__':
                                          arguments['<changed_value>'])
             logger.debug(inventory.data)
             inventory.save()
-    if arguments['manager'] is True:
-        logger.debug("Manager commands")
-        if arguments['getinfo'] is True:
-            logger.debug('getinfo command')
-            # If manager is not defined set it to 'default'
-            if not arguments['<manager_name>']:
-                manager_name = 'default'
-            else:
-                manager_name = arguments['<manager_name>']
-            # Check if the default section is available in our conf file
-            inventory.check_manager(manager_name)
-            if arguments['--insecure'] is True:
-                get_manager_info(manager_name, False)
-            else:
-                get_manager_info(manager_name, True)
+    elif arguments['getinfo'] is True:
+        logger.debug('getinfo command')
+        # If manager is not defined set it to 'default'
+        if not arguments['<manager_name>']:
+            manager_name = 'default'
+        else:
+            manager_name = arguments['<manager_name>']
+        # Check if the default section is available in our conf file
+        inventory.check_manager(manager_name)
+        connection_parameters = inventory.get_manager_info(manager_name)
 
+        print('Gathering data from manager, please wait...\n')
+        # TODO : Add a rotating star showing program is running ?
+        #        Could be a nice exercice for learning python. :)
+        logger.info('Gathering data from manager')
+
+        if arguments['--insecure'] is True:
+            redfish_data = get_redfish_data(connection_parameters, False)
+        else:
+            redfish_data = get_redfish_data(connection_parameters, True)
```

Comment:
Could be redfish_data = get_redfish_data(connection_parameters, ! arguments['--insecure'])